DigitalOffice/Services/ProjectService/Mappers/ProjectMapper.cs

```diff
@@ -35,6 +35,24 @@ public DbProject Map(NewProjectRequest request)
                 DepartmentId = request.DepartmentId,
                 Description = request.Description,
                 IsActive = request.IsActive,
+                Name = request.Name,
+                Deferred = false
+            };
+        }
+
+        public DbProject Map(EditProjectRequest request)
+        {
+            if (request == null)
+            {
+                throw new ArgumentNullException(nameof(request));
+            }
+
+            return new DbProject
+            {
+                DepartmentId = (Guid)request.DepartmentId,
```

Comment:
What happens if there is null?

Reply:
It can't map nullable to not nullable. So there were Build errors

Reply:
Is this normal practice?